ovs-p4rt/include/ovsp4rt/ovs-p4rt.h

```diff
@@ -86,7 +86,7 @@ struct tunnel_info {
   struct p4_ipaddr local_ip;
   struct p4_ipaddr remote_ip;
   uint16_t dst_port;  // bit<16>
-  uint16_t vni;       // bit<24>
+  uint32_t vni;       // bit<24>
```

Comment:
What are the repurcussions of changing this to 32-bit here and not in OVS codebase? 

I understand that there is a copy of the ovs-p4rt.h in OVS repo, which still has the uint16.

Further, even if we update that OVS copy to uin32 - need to make sure that OVS can use the corrected value and all references are checked

Reply:
I haven't submitted the OVS PR yet. Thanks to one or more recent software updates, WSL's connection to the outside network goes dead and I have to reboot to recover. This leads to delays -- sometimes long ones, because it's late in the day or I'm frustrated and need to take a break -- and I frequently forget, when I return to my computer, that I have an unpushed PR to return to. That's what happened in this case. And when I found it this morning, and went to push it ... you guessed. The network connection was dead again. So I broke for lunch.

Reply:
> What are the repercussions of changing this to 32-bit here and not in OVS codebase?

Not an issue if OVS is built `--with-ovsp4rt`. OvS uses the copy in the networking-recipe install tree.

Bad if it's built with `--with-p4ovs`. OvS uses its private copy.